contracts/Cloudwalk/CErc20.sol

```diff
@@ -150,9 +155,11 @@ contract CErc20 is CToken, CErc20Interface {
      *      Note: This wrapper safely handles non-standard ERC-20 tokens that do not return a value.
      *            See here: https://medium.com/coinmonks/missing-return-value-bug-at-least-130-tokens-affected-d67bf08521ca
      */
-    function doTransferIn(address from, uint amount) internal returns (uint) {
-        EIP20NonStandardInterface token = EIP20NonStandardInterface(underlying);
-        uint balanceBefore = EIP20Interface(underlying).balanceOf(address(this));
+    function doTransferIn(address from, uint amount) virtual override internal returns (uint) {
+        // Read from storage once
+        address underlying_ = underlying;
```

Comment:
why do we need to do this attribution to a new variable>

Reply:
It's cheaper to initiate a local variable and read from it multiple times than reading from the storage variable multiple times.